.travis.yml

```diff
@@ -24,19 +22,17 @@ install:
   - wget -qO- get.nextflow.io | bash
   - sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
   # Install nf-core/tools
-  - git clone https://github.com/nf-core/tools.git /tmp/nf-core-tools
-  - cd /tmp/nf-core-tools
-  - pip install --user -e .
+  - pip install nf-core
   # Reset
-  - mkdir ${TRAVIS_BUILD_DIR}/tests
-  - cd ${TRAVIS_BUILD_DIR}/tests
+  - mkdir ${TRAVIS_BUILD_DIR}/tests && cd ${TRAVIS_BUILD_DIR}/tests
 
 env:
-  - NXF_VER=0.30.0 # Specify a minimum NF version that should be tested and work
+  - NXF_VER='0.32.0' # Specify a minimum NF version that should be tested and work
   - NXF_VER='' # Plus: get the latest NF version and check, that it works
 
 script:
   # Lint the pipeline code
   - nf-core lint ${TRAVIS_BUILD_DIR}
   # Run the pipeline with the test profile
-  - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker
+  # - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --read 'data/reads_{1,2}.fq.gz' --genome GRCh38
+  - docker run -it -v ${TRAVIS_BUILD_DIR}:/${TRAVIS_BUILD_DIR} mproksik/rnafusion-test /bin/bash -c 'nextflow run ${TRAVIS_BUILD_DIR} -profile test --reads "/test-data/test_{1,2}.fastq.gz" --genome R64-1-1 --star_fusion'
```

Comment:
should `mproksik/rnafusion-test` be `nf-core/rnafusion-test` instead?

Reply:
This is super cool 👍